src/form.js

```diff
@@ -125,7 +125,8 @@ define([
 
         this.formName = 'New Form';
         this.mugMap = {};
-        this.hashtagDictionary = {};
+        this.hashtagMap = {};
+        this.invertedHashtagMap = {};
```

Comment:
Renamed `hashtagDictionary` to `hashtagMap` because `invertedHashtagDictionary` was way too long.